Draggable.js

```diff
@@ -150,11 +150,14 @@ export default class Draggable extends Component {
 		};
 	}
 	_getTextOrImage = () => {
-		const { renderSize, renderShape, renderText, imageSource } = this.props;
+		const { renderSize, renderShape, renderText, imageSource, children } = this.props;
 		if(renderShape == 'image') {
 			return(<Image style={this._dragItemCss(renderSize, null, 'image')} source={imageSource}/>);
 		}else{
-			return (<Text style={this._dragItemTextCss(renderSize)}>{renderText}</Text>);
+			if(children===null)
+				return (<Text style={this._dragItemTextCss(renderSize)}>{renderText}</Text>);
+			else
+				return (children)
```

Comment:
160:22  warning  Missing semicolon  semi

✖ 1 problem (0 errors, 1 warning)

Reply:
@vigilantee 
hello vigilantee,
I would like to merge this pull request if you can adjust your code and run "npm run lint" to check the coding style. 

I might change the children to rendeChildren in order to follow our api naming style.
anyway, I will update ReadMe and organize it.